### PR TITLE
styling home page #32

### PR DESCRIPTION
**Pull request by MahmoudMH**
#32
make styling for home page .
- [x] hard code at the logo Css>> so we need to solve .

### CODE REVIEW

File: public/home.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  rel="stylesheet">
+  <link rel="stylesheet" href="css/home.css">
+  <link rel="shortcut icon" type="image/x-icon" href="/image/icon.ico" />
+  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display" rel="stylesheet">
+</head>
+  <body>
+    <header class="header">
+      <form class="headerForm" action="" method="post">
+        <input class="input" type="submit" name="" value="Login">
```
Comment (walaamedhat) [inline public/home.html]: give input name

**Review by walaamedhat — CHANGES_REQUESTED** [review]
:+1:

**Review by Salam-Dalloul — DISMISSED** [review]

**Review by walaamedhat — APPROVED** [review]

**Review by Salam-Dalloul — DISMISSED** [review]

**Review by walaamedhat — DISMISSED** [review]

**Review by muhmushtaha — COMMENTED** [review]
what about the responsive?

**Comment by MahmoudMH** [comment]
@muhmushtaha 
it's already responsive ^_^

**Review by muhmushtaha — APPROVED** [review]
Still need some work @MahmoudMH

**Pull request merged**